README.md

```diff
@@ -22,58 +22,60 @@ You first need to build the cli before building the docker images.
 
 ### Base image
 
-If you make changes to the [`src`](./src/) folder or the [`Dockerfile`](./Dockerfile), you need to rebuild the `containerbase/base` image.
+If you make changes to the [`src`](./src/) folder or the [`Dockerfile`](./Dockerfile), you must:
+
+1. run `pnpm build`
+1. rebuild the `containerbase/base` image
 
 ```sh
 docker buildx bake
 ```
 
-You can use the following command to ignore remote cache for local testing.
-This will probably speedup local builds.
+You can use the following command to ignore the remote cache for local testing.
+This may speed up your local builds.
 
 ```sh
 docker buildx bake  --set *.cache-from=
 ```
 
 ### Test images
 
-To run one of the tests use the following command, it will run the java tests from [`test/java`](./test/java/).
+To run one of the tests use the following command, it will run the Java tests from [`test/java`](./test/java/).
 
 ```sh
 TAG=java docker buildx bake test
```

Comment:
I'll make new suggestions tomorrow, there's a simpler pnpm command now

Reply:
You can run `pnpm test:docker -b -t java` instead. the `-b` causes the required binaries to rebuild before testing.

`pnpm test:docker` causes all language tests to run.

Reply:
Can you suggest the changes you want?

Reply:
let's leave it as is for now. it's not wrong.